wmi_check/assets/configuration/spec.yaml

```
@@ -0,0 +1,113 @@
+name: WMI
+files:
+- name: wmi_check.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/default
+
+  - template: instances
+    options:
+    - name: class
+      required: true
+      description: |
+        `class` is the name of the WMI class, for example `Win32_OperatingSystem`.
+        You can find many of the standard class names on the MSDN docs:
+
+        https://msdn.microsoft.com/en-us/library/windows/desktop/aa394084.aspx
+      value:
+        type: string
+    - name: metrics
+      required: true
+      description: |
+        list of metrics you want to capture, with each item in the list being a set of:
+        [<WMI_PROPERTY_NAME>, <METRIC_NAME>, <METRIC_TYPE>]
+
+          * <WMI_PROPERTY_NAME> is something like `NumberOfUsers` or `ThreadCount`.
+                                The standard properties are also available on the MSDN docs
+                                for each class.
+
+          * <METRIC_NAME> is the name you want to show up in Datadog.
+
+          * <METRIC_TYPE> is from the standard choices for all agent checks, such as gauge, rate, histogram or counter.
```

Seems like an extra space sneaked in:

(Also IMO the indentation on the description of `WMI_PROPERTY_NAME` looks weird, I know it was there before but maybe we could fix it?)

```suggestion
         * <WMI_PROPERTY_NAME> is something like `NumberOfUsers` or `ThreadCount`.
                                The standard properties are also available on the MSDN docs
                                for each class.

         * <METRIC_NAME> is the name you want to show up in Datadog.

         * <METRIC_TYPE> is from the standard choices for all agent checks, such as gauge, rate, histogram or counter.
```